src/functional/resources/repo-config.csv

```diff
@@ -1,5 +1,5 @@
-Repository's Location,Branch,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Alpha.git,master,,
-https://github.com/reposense/testrepo-Beta.git,master,docs**,yes
-https://github.com/reposense/testrepo-Charlie.git,master,,
-https://github.com/reposense/testrepo-Delta.git,master,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config
+https://github.com/reposense/testrepo-Alpha.git,master,,,
+https://github.com/reposense/testrepo-Beta.git,master,java;adoc,docs**,yes
+https://github.com/reposense/testrepo-Charlie.git,master,java;adoc,,
+https://github.com/reposense/testrepo-Delta.git,master,java;adoc,,
```

Comment:
is your feature working? how is it that adding the formats `java;adoc` to 3 of the test repos, yet the functional still pass without change to the generated files?

Reply:
Because functional tests was using `java` and `adoc` as formats.